### PR TITLE
Move pdf viewer reducer

### PR DESCRIPTION
**Pull request by tejans24**
connects #2829

### CODE REVIEW

File: client/app/reader/reducer.js [hunk 1]
```diff
@@ -64,115 +19,12 @@ export const initialState = {
...
-    );
-  case Constants.RECEIVE_APPEAL_DETAILS:
-    return update(state,
-      {
-        loadedAppeal: {
-          $set: action.payload.appeal
-        }
-      }
-    );
-  case Constants.RECEIVE_APPEAL_DETAILS_FAILURE:
```
Comment (mdbenjam) [inline client/app/reader/reducer.js]: This no longer works. I think we need to change where we use: didLoadAppealFail. We're still referencing it from the readerReducer.

File: client/app/reader/PdfSidebar.jsx [hunk 2]
```diff
@@ -256,10 +257,10 @@ const mapStateToProps = (state, ownProps) => {
     ..._.pick(state.annotationLayer, 'placedButUnsavedAnnotation', 'selectedAnnotationId'),
     comments: makeGetAnnotationsByDocumentId(state)(ownProps.doc.id),
     scrollToSidebarComment: state.readerReducer.ui.pdf.scrollToSidebarComment,
```
Comment (mdbenjam) [inline client/app/reader/PdfSidebar.jsx]: this is now in a new reducer.

File: client/app/reader/PdfViewer/PdfViewerReducer.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        $set: uniqueTags
+      }
+    });
+  case Constants.TOGGLE_PDF_SIDEBAR:
+    return update(state, {
+      hidePdfSidebar: { $set: !state.ui.pdf.hidePdfSidebar } }
+    );
+  case Constants.SET_DOC_SCROLL_POSITION:
+    return update(state, {
+      scrollTop: { $set: action.payload.scrollTop }
```
Comment (mdbenjam) [inline client/app/reader/PdfViewer/PdfViewerReducer.js]: You're still referencing the old scrollTop in the old reducer.

File: client/app/reader/reducer.js [hunk 4]
```diff
@@ -64,115 +19,12 @@ export const initialState = {
...
-          scrollTop: { $set: action.payload.scrollTop }
-        }
-      }
-    });
   case Constants.SCROLL_TO_COMMENT:
     return update(state, {
-      ui: { pdf: { scrollToComment: { $set: action.payload.scrollToComment } } }
+      scrollToComment: { $set: action.payload.scrollToComment }
     });
-  case Constants.TOGGLE_PDF_SIDEBAR:
```
Comment (mdbenjam) [inline client/app/reader/reducer.js]: This is not working.

**Comment by tejans24** [comment]
Jump to comment isn't working right now. I am trying to fix it.

**Pull request merged**